getting-started/create-a-mobile-property.md

```diff
@@ -21,26 +21,26 @@ The default privacy status is set to _opted in_ and might impact data collection
 
 ## Set up your extensions
 
-1. In Experience Platform Launch, click the **Extensions** tab. The **Mobile Core** and **Profile** extensions are installed by default.
+1. Within the mobile property, click the **Extensions** tab. The **Mobile Core** and **Profile** extensions are installed by default and will display on the tab.
```

Comment:
- Please replace "Within" with "In"
- Please move the 2nd line ("The **Mobile Core** and...) to a new line like this:
"1. In the Mobile property, click the **Extension** tab.
        The **Mobile Core** and **Profile** extensions are installed by default and will not display on the tab."
(The start of the sentence should line up with the start of the step text.)